### PR TITLE
Add option configure ports mapping

### PR DESCRIPTION
**Pull request by lukas-bednar**

### CODE REVIEW

**Comment by lukas-bednar** [comment]
Hm ... there is some problem to install packages for centos7, maybe some mirror problem.
However it works for me ok, I will try to retrigger that build again later ...

https://travis-ci.org/chrismeyersfsu/provision_docker/builds/242375361

File: test/playbook_verify_ports.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+- name: Verify that sevice is exposed on host's port
+  hosts: localhost
+  tasks:
+    - name: Wait for nginx appear on port 8080
+      uri:
+        url: "http://localhost:8080"
+        status_code: 200
+      register: result
+      until: result.status == 200  # Welcome page is not really welcoming
```
Comment (lukas-bednar) [inline test/playbook_verify_ports.yml]: Remove comment, it is not relevant with nginx apache ...

**Comment by lukas-bednar** [comment]
From some reason I can not install "anything" from outside of container when setting `ports` field.
This issue happens on travis-ci, and I wasn't able to reproduce that issue locally.
I tried different images (centos7, ubuntu14.04, 18.04) and all of them worked on local system and failed on travis.

I don't have any idea what else to try, any thoughts are welcomed.

**Comment by haidaraM** [comment]
Mine is working properly when I'm testing my role https://github.com/haidaraM/ansible-role-django/blob/tests/test/test.yml#L34

You can see my travis build here: https://travis-ci.org/haidaraM/ansible-role-django/builds/313528004

**Comment by lukas-bednar** [comment]
@haidaraM thanks for hint, and good news, I was able to reproduce that issue on my local env. so now I think I can move on.

**Pull request closed without merging**